pkg/client/cli/cmds_intercept.go

```diff
@@ -56,10 +57,27 @@ type interceptArgs struct {
 	cmdline []string // Args[1:]
 }
 
+// safeCobraCommand is more-or-less a subset of *cobra.Command, with less stuff exposed so I don't
+// have to worry about things using it in ways they shouldn't.
+type safeCobraCommand interface {
+	InOrStdin() io.Reader
+	OutOrStdout() io.Writer
+	ErrOrStderr() io.Writer
+	FlagError(error) error
+}
+
+type safeCobraCommandImpl struct {
+	*cobra.Command
+}
+
+func (w safeCobraCommandImpl) FlagError(err error) error {
+	return w.Command.FlagErrorFunc()(w.Command, err)
+}
+
```

Comment:
I dropped the commit moving this to cliutil, since changing `Run` to just use `os.Std{in,out,err}` made moving it unnecessary.

https://github.com/telepresenceio/telepresence/pull/1864#discussion_r681944818